students/jay_ferguson/lesson03/mailroom.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python3
+
+from cli import CLI
+
+cli = CLI()
+
+donors = [
```

Comment:
This data structure would have performance issues if you were searching by donor's name. That's why we would have used dictionary with name as a key. Also, the point of the exercise was to CALCULATE the total and the number of gifts based on particular gifts.